### PR TITLE
feature: use woodpecker ci instead of drone

### PR DESCRIPTION
**Pull request by Jan-PieterBaert**

### CODE REVIEW

File: .woodpecker/.test-pr.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  lint:
+    image: danlynn/ember-cli:4.8.0
+    group: lint
+    commands:
+      - npm run lint
+  test:
+    image: danlynn/ember-cli:4.8.0
+    commands:
+      - npm run test:ember
+    failure: ignore
```
Comment (Windvis) [inline .woodpecker/.test-pr.yml:15]: Was there a specific reason why this was added? Or copy-pasta from a different setup?

```suggestion
```

File: .woodpecker/.latest.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+steps:
+  build:
+    image: woodpeckerci/plugin-docker-buildx
+    settings:
+      repo: ${CI_REPO}
+      tags: latest
+    secrets: [ docker_username, docker_password ]
+when:
+  branch: master
```
Comment (Windvis) [inline .woodpecker/.latest.yml:9]: It seems this was set to 'development' in the drone version, I would keep it to make the behavior consistent.
```suggestion
  branch: development
```

File: .woodpecker/.test-pr.yml (hunk 1 above)
Comment (Jan-PieterBaert) [reply .woodpecker/.test-pr.yml:15]: This was indeed a copy-paste problem

File: .woodpecker/.test-pr.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+steps:
+  install:
+    image: danlynn/ember-cli:4.8.0
+    commands:
+      - npm ci
```
Comment (Windvis) [inline .woodpecker/.test-pr.yml:5]: It seems this project doesn't use lock files yet.

```suggestion
      - npm install
```

**Comment by Windvis** [comment]
The dependency issue will be resolved by #26 so merging this already. Thanks!

**Pull request merged**